aspnetcore/release-notes/aspnetcore-3.0.md

```diff
@@ -0,0 +1,478 @@
+---
+title: What's new in ASP.NET Core 3.0
+author: rick-anderson
+description: Learn about the new features in ASP.NET Core 3.0.
+ms.author: riande
+ms.custom: mvc
+ms.date: 09/17/2019
+uid: aspnetcore-3.0
+---
+# What's new in ASP.NET Core 3.0
+
+This article highlights the most significant changes in ASP.NET Core 3.0 with links to relevant documentation.
+
+## ASP.NET Core 3.0 only runs on .NET Core 3.0
+
+As of ASP.NET Core 3.0, .NET Framework is no longer a supported target framework. Projects targeting .NET Framework can continue in a fully supported fashion using the [.NET Core 2.1 LTS release](https://www.microsoft.com/net/download/dotnet-core/2.1). ASP.NET Core 2.1 is supported until August 21, 2021. For more information, see [this GitHub announcement](https://github.com/aspnet/Announcements/issues/324).
+
+See [Port your code from .NET Framework to .NET Core](/dotnet/core/porting/) for migration information.
+
+## Use the ASP.NET Core shared framework
+
+The [ASP.NET Core 3.0 shared framework](https://natemcmaster.com/blog/2018/08/29/netcore-primitives-2/), contained in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app), no longer requires an explicit `<PackageReference />` element in the project file. The shared framework is automatically referenced when using the `Microsoft.NET.Sdk.Web` SDK in your project file:
```

Comment:
Let's not link to third party blog posts from official docs that we don't have control over.

```suggestion
The ASP.NET Core 3.0 shared framework, contained in the [Microsoft.AspNetCore.App metapackage](xref:fundamentals/metapackage-app), no longer requires an explicit `<PackageReference />` element in the project file. The shared framework is automatically referenced when using the `Microsoft.NET.Sdk.Web` SDK in your project file:
```